modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js

```diff
@@ -72,6 +72,13 @@ export default {
         return 'indeterminate';
       }
       return 'empty';
+    },
+    // Default implementation of isModified is based on whether the
+    // selection has changed, but you can override this and combine
+    // that bit with your own if your manager allows in-context editing
+    // of a piece (i.e. AposMediaManager)
+    isModified() {
```

Comment:
Should this be computed? It's based off a method, not other properties.

Reply:
That change has in fact happened.

Reply:
I'm questioning that it's a good idea that it be computed since it does not include the properties on which it depends -- only a method to execute.

Reply:
A computed property function does not have to touch reactive properties itself. It may do it indirectly, by calling a method that examines reactive properties, and so on.